packages/shared/src/components/modals/common/ModalFooter.tsx

```diff
@@ -16,10 +16,16 @@ export function ModalFooter({
   justify = Justify.End,
   view,
 }: ModalFooterProps): ReactElement {
-  const { activeView, isDrawer } = useContext(ModalPropsContext);
-  if ((view && view !== activeView) || isDrawer) {
+  const { activeView, isDrawer, isForm } = useContext(ModalPropsContext);
+
+  if (isForm || isDrawer) {
```

Comment:
@capJavert let's see if anyone objects 😂

Reply:
Just generic so we have the following options now:
- modal
-- is it drawer?
-- is it form?

Sounds like mayhem 😅